resources/css/components/tabs.css

```diff
@@ -1,17 +1,37 @@
 /* Tabs */
 .tabs {
+  @apply overflow-hidden;
   &-list {
     @apply flex gap-2 overflow-x-auto border-b border-gray-200 dark:border-dark-200;
   }
-
+  &-title{
+    @apply flex gap-2 w-full mr-2 text-primary;
+  }
   &-button {
     @apply flex h-10 items-center gap-1.5 whitespace-nowrap border-b border-b-transparent px-2 text-3xs font-medium text-gray-400 hover:border-gray-300 hover:text-gray-700 current:pointer-events-none current:border-primary current:text-primary dark:text-slate-400 dark:hover:border-gray-500 dark:hover:text-slate-300 dark:current:border-secondary dark:current:text-secondary sm:h-12 sm:text-2xs;
     > svg {
       @apply h-4 w-4 sm:h-5 sm:w-5;
     }
   }
-
   &-content {
-    @apply mt-6;
+    @apply mt-6 overflow-hidden;
+  }
+  &-vertical {
+    @apply flex gap-3;
+    .tabs-list {
+      @apply flex flex-col min-h-max border-r border-b-0 min-w-max;
+    }
+    .tabs-button{
+        @apply w-full border-r border-r-transparent border-b-0;
+        &._is-active{
+            @apply border-primary
+        }
+        &:hover{
+            @apply border-gray-300
+        }
+    }
+    .tabs-content{
+        @apply grow;
```

Comment:
<img width="692" alt="Снимок экрана 2024-05-21 в 08 17 57" src="https://github.com/moonshine-software/moonshine/assets/12373059/8477af6d-b96d-4050-8a0a-c6540102a07f">

```suggestion
        @apply grow mt-3;
```

<img width="690" alt="Снимок экрана 2024-05-21 в 08 19 24" src="https://github.com/moonshine-software/moonshine/assets/12373059/b9f36e69-5517-41b7-97a3-3f0174b246fe">